File: 1470-shuffle-the-array/1470-shuffle-the-array.c
/**
 * Note: The returned array must be malloced, assume caller calls free().
 */
int* shuffle(int* nums, int numsSize, int n, int* returnSize){
    int* final = (int*)malloc(sizeof(int) * numsSize);
    int i = 0;
    int m = 0;
    while(m < n){
        final[i++] = nums[m];
        final[i++] = nums[m + n];
        m ++;
    }
    *returnSize = numsSize;
    return final;
}

File: 1470-shuffle-the-array/1470-shuffle-the-array.py
class Solution(object):
    def shuffle(self, nums, n):
        """
        :type nums: List[int]
        :type n: int
        :rtype: List[int]
        """
        f = []
        for i in range(n):
            f.append(nums[i])
            f.append(nums[n + i])
        return f